src/app/funding/funding.component.ts

```diff
@@ -1,10 +1,75 @@
 import { Component } from '@angular/core';
 
+export interface Funder {
+  title: string;
+  website: string;
+  imageSource: string;
+  altImageText: string;
+  content: string;
+}
+
 @Component({
   selector: 'app-funding',
   templateUrl: './funding.component.html',
   styleUrls: ['./funding.component.scss']
 })
 export class FundingComponent {
+  GenomeCanadaFunder: Funder = {
+    title: 'Genome Canada',
+    website: 'https://www.genomecanada.ca/',
+    imageSource: '../assets/images/sponsors/genomecanada.png',
+    altImageText: 'Genome Canada Logo',
+    content: `Genome Canada is a not-for-profit organization, funded by the Government of Canada. We act as a catalyst for developing and
+    applying genomics+ and genomic-based technologies to create economic and social benefits for Canadians.`
+  };
+  ProvinceOfOntarioFunder: Funder = {
+    title: 'Province of Ontario',
+    website: 'https://www.ontario.ca',
+    imageSource: '../assets/images/sponsors/ontario_new.png',
+    altImageText: 'Government of Ontario Logo',
+    content: 'This work was funded by the Government of Canada through Genome Canada and the Ontario Genomics Institute (OGI-168).'
+  };
+  NIHFunder: Funder = {
+    title: 'National Institutes of Health',
+    website: 'https://www.nih.gov/',
+    imageSource: '../assets/images/sponsors/NIH.png',
+    altImageText: 'NIH Logo',
+    content: 'A part of the U.S. Department of Health and Human Services, NIH is the largest biomedical research agency in the world.'
+  };
+  BISTIFunder: Funder = {
+    title: 'BISTI',
+    website: 'https://www.nih.gov/',
+    imageSource: '../assets/images/sponsors/bisti-logo.png',
+    altImageText: 'BISTI Logo',
+    content: `The Biomedical Information Science and Technology Initiative is a consortium of representatives from each of the NIH
+    institutes and centers. The mission of BISTI is to make optimal use of computer science and technology to address problems in
+    biology and medicine by fostering new basic understandings, collaborations, and transdisciplinary initiatives between the
+    computational and biomedical sciences.`
+  };
+  DataSTAGEFunder: Funder = {
+    title: 'DataSTAGE',
```

Comment:
Let's change DataStage to Biodata Catalyst as per https://oicr.slack.com/archives/C16ET3CF4/p1585331991046100

Will need new site, logo, content